site/DataInsightsIndexPageContent.tsx

```diff
@@ -88,15 +89,19 @@ export const DataInsightsIndexPageContent = (
 ) => {
     const { pageNumber, dataInsights, isPreviewing = false } = props
     // Extract all attachments from the data insights and supply them to the AttachmentsContext
-    const { imageMetadata, linkedCharts, linkedDocuments } =
+    const { imageMetadata, linkedAuthors, linkedCharts, linkedDocuments } =
         dataInsights.reduce(
             (acc, di) => ({
                 imageMetadata: merge(acc.imageMetadata, di.imageMetadata),
+                linkedAuthors: di.linkedAuthors
```

Comment:
While working on this, did you see anything that explained why linkedAuthors should be an array instead of a dictionary? Seems like we don't need order information, and it's resulted in a bit of extra boilerplate. 🤔

Reply:
I'm not sure what you mean. Semantically linked authors are a collection of entities, not a mapping and while it's true their order doesn't matter, I wouldn't want to force it into an object to save a few lines of boilerplate.

But as I wrote this, I realized we should at least dedupe the array, so I'm adding that.

Reply:
Right, but given that we don't appear to need to iterate through them at any point, I'm wondering why the decision was made to store them in an array as opposed to dictionaries like we've done for all other attachments (I was on a break when the feature was developed)

I'm not suggesting you change it, I was just wondering if during this work you saw something that justified it that I've missed 🙂